front/app/containers/IdeasEditPage/IdeasEditForm.tsx

```diff
@@ -196,7 +196,6 @@ const IdeasEditForm = ({ ideaId }: Props) => {
       data.idea_images_attributes ||
       Object.values(data.body_multiloc).some((value) => value.includes('<img'));
 
-    setInitialFormData(data);
     setFormData(data);
```

Comment:
Looks like we were setting the "latest form data" to the "initial form data" attribute before submitting, which caused the `isImageNew` check to not work correctly submitting, so we weren't deleting images correctly.

If there's a specific reason we do this `setInitialFormData(data)` though which I've missed, just let me know!